app/src/main/java/com/nervousfish/nervousfish/activities/FirstUseActivity.java

```diff
@@ -19,22 +17,14 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger("FirstUseActivity");
 
-    private IServiceLocator serviceLocator;
-
     /**
-     * Creates the new activity, should only be called by Android.
-     *
-     * @param savedInstanceState The saved state of the previous instance.
+     * {@inheritDoc}
      */
     @Override
     protected void onCreate(final Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         this.setContentView(R.layout.activity_first_use);
-
-        final Intent intent = this.getIntent();
-        this.serviceLocator = (IServiceLocator) intent.getSerializableExtra(ConstantKeywords.SERVICE_LOCATOR);
-
-        LOGGER.info("FirstUseActivity created");
+        LOGGER.info("activity created");
```

Comment:
Indeed I did 🙂 This avoid incorrect logs when people copy the `onCreate` method to other activities